index.html

```diff
@@ -0,0 +1,359 @@
+
+        <ul class="nav-links">
+          <li class="link">
+            <a class="responsive_link" href="#">Who We Are</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">What We Do</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">Travel With Us</a>
+          </li>
+          <li class="link">
+            <a class="responsive_link" href="#">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <h1>Bhromaon</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12">
+            <p>Safe Travel Anywhere</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12">
+            <div class="search">
+              <input
+                type="search"
+                placeholder="Search Your Travel Destination"
+                name="q"
+              />
+              <button name="submit" class="header_button" type="submit">
+                <span class="icon-search"></span> Search
+              </button>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12">
+            <p class="scroll">Scroll Down</p>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-12"><span class="icon-chevron-down"></span></div>
+        </div>
+      </div>
+    </header>
+    <section class="nav">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-2 col-md-12">
+            <p>Bhromaon</p>
+          </div>
+          <div class="col-lg-10 col-md-12">
+            <ul class="menu-main">
+              <li><a href="#" class="first_link">Who We Are</a></li>
+              <li><a href="#">What We Do</a></li>
+              <li><a href="#">Travel With Us</a></li>
+              <li><a href="#">Contact Us</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="WhyChooseUs">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-6 col-md-12 reverse">
+            <div class="row">
+              <div class="col-12"><h2>Why Choose Us</h2></div>
+              <div class="col-12">
+                <p>
+                  Tion and dislike men who are so beguiled and demoralized by
+                  desire by the charms of pleasure of the moment, so blinded by
+                  desire, that they cannot foresee Ddislike men who are so
+                  beguiled and demoralized by the charms of pleasure of the
+                  moment, so blinded by desire, that.
+                </p>
+              </div>
+              <div class="col-12 WhyChooseUs_button">
+                <a href="#" class="button">Explore Places </a>
+              </div>
```

Comment:
Старайся использовать БЭМ
```suggestion
            <div class="why-choose-us__content">
              <h2 class="why-choose-us__title">Why Choose Us</h2>
              <p class="why-choose-us__text">
                  Tion and dislike men who are so beguiled and demoralized by
                  desire by the charms of pleasure of the moment, so blinded by
                  desire, that they cannot foresee Ddislike men who are so
                  beguiled and demoralized by the charms of pleasure of the
                  moment, so blinded by desire, that.
                </p>
              <a href="#" class="button why-choose-us__button">Explore Places </a>
            </div>
```